javascript-forms/form-values.js

```diff
@@ -0,0 +1,13 @@
+var elContactForm = document.getElementById('contact-form');
+var newObject = {};
```

Comment:
You should make this variable inside of the function block, since you don't want it to continue to exist after the function has ended.